src/graphql/Search.js

```diff
@@ -1,4 +1,6 @@
-query($query:String!, $pageCursor:String) {
+import gql from 'graphql-tag';
+
+export default gql`query($query:String!, $pageCursor:String) {
```

Comment:
Now that this isn't doing whatever invisible magic it was doing by having the `.gql` extension, we have to use the very common and highly documented `graphql-tag` to declare this as a graphql query. All the official docs say to do this, so seems sensible to do.